designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsService.scala

```diff
@@ -143,25 +120,25 @@ class UsageStatisticsReportsSettingsService(
     fetchFeStatistics: () => Future[Map[String, Long]]
 )(implicit ec: ExecutionContext) {
 
-  def prepareStatisticsUrl(): Future[Either[StatisticError, Option[URL]]] = {
+  def prepareStatisticsUrl(): Future[Either[StatisticError, List[URL]]] = {
```

Comment:
WDYT about changing the code to not assume we are in the URL context here? 
All the class does is collect data and create statistics. The conversion from the Statistics to the URL can be done at the end (an URL is a representation of the Nu stat and could be understand as a form of the stats serialization)